features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,635 @@
+      | naam                                 | waarde                        |
+      | voornamen                            | Hendrik Jan                   |
+      | adellijkeTitelPredicaat.code         | R                             |
+      | adellijkeTitelPredicaat.omschrijving | Ridder                        |
+      | adellijkeTitelPredicaat.soort        | titel                         |
+      | voorvoegsel                          | de                            |
+      | geslachtsnaam                        | Brink                         |
+      | aanduidingNaamgebruik.code           | E                             |
+      | aanduidingNaamgebruik.omschrijving   | gebruik alleen de eigen naam. |
+
+
+  Rule: wanneer gevraagd wordt om aanhef, aanschrijfwijze of gebruikInLopendeTekst, dan worden naast het geslacht en de naamgegevens van de persoon ook de naamgegevens van de partner geleverd
+    - van de partner worden adellijkeTitelPredicaat, voorvoegsel en geslachtsnaam geleverd
+   
+    Abstract Scenario: met fields is gevraagd om <veld>
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000176                       |
+      | fields              | <veld>                          |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                                      | waarde                        |
+      | naam.voornamen                            | Hendrik Jan                   |
+      | naam.adellijkeTitelPredicaat.code         | R                             |
+      | naam.adellijkeTitelPredicaat.omschrijving | Ridder                        |
+      | naam.adellijkeTitelPredicaat.soort        | titel                         |
+      | naam.voorvoegsel                          | de                            |
+      | naam.geslachtsnaam                        | Brink                         |
+      | naam.aanduidingNaamgebruik.code           | E                             |
+      | naam.aanduidingNaamgebruik.omschrijving   | gebruik alleen de eigen naam. |
+      | geslacht.code                             | M                             |
+      | geslacht.omschrijving                     | man                           |
+      En heeft de response een persoon met een 'partner' met de volgende gegevens
+      | naam                                      | waarde  |
+      | naam.adellijkeTitelPredicaat.code         | BS      |
+      | naam.adellijkeTitelPredicaat.omschrijving | Barones |
+      | naam.adellijkeTitelPredicaat.soort        | titel   |
+      | naam.voorvoegsel                          | van der |
+      | naam.geslachtsnaam                        | Horn    |
+
+      Voorbeelden:
+      | veld                             |
+      | aanhef                           |
```

Comment:
voor aanhef heeft de brpproxy ook partners.aangaanHuwelijkPartnerschap.datum en partners.ontbindingHuwelijkPartnerschap.datum.

Reply:
Ik neem aan dat je bedoelt dat je dit nodig hebt.

Is toegevoegd.